html/elements/input/password.json

```diff
@@ -0,0 +1,1070 @@
+                },
+                "edge": {
+                  "version_added": null
+                },
+                "edge_mobile": {
+                  "version_added": null
+                },
+                "firefox": {
+                  "version_added": "4"
+                },
+                "firefox_android": {
+                  "version_added": null
+                },
+                "ie": {
+                  "version_added": "10"
+                },
+                "opera": {
+                  "version_added": "11"
+                },
+                "opera_android": {
+                  "version_added": null
+                },
+                "safari": {
+                  "version_added": null
+                },
+                "safari_ios": {
+                  "version_added": null
+                },
+                "webview_android": {
+                  "version_added": null
+                }
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "formmethod": {
+            "__compat": {
+              "description": "<code>formmethod</code> attribute",
+              "support": {
+                "chrome": {
+                  "version_added": "9"
+                },
+                "chrome_android": {
+                  "version_added": null
+                },
+                "edge": {
+                  "version_added": null
+                },
+                "edge_mobile": {
+                  "version_added": null
+                },
+                "firefox": {
+                  "version_added": "4"
+                },
+                "firefox_android": {
+                  "version_added": "4"
+                },
+                "ie": {
+                  "version_added": "10"
+                },
+                "opera": {
+                  "version_added": "11"
+                },
+                "opera_android": {
+                  "version_added": "11"
+                },
+                "safari": {
+                  "version_added": "6"
+                },
+                "safari_ios": {
+                  "version_added": "5"
+                },
+                "webview_android": {
+                  "version_added": null
+                }
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "formmnovalidate": {
+            "__compat": {
+              "description": "<code>formnovalidate</code> attribute",
+              "support": {
+                "chrome": {
+                  "version_added": "5",
+                  "notes": "In 6.0 it only worked with the HTML5 document type, validation support in 7.0 was disabled and re-enabled in 10.0."
```

Comment:
It seems to me like this would be easier to understand as an array of entries:

- Version added 10
- Version added 5, version removed 7, with a note about Chrome 6 HTML5 doctype restriction